client/my-sites/site-metrics/controller.tsx

```diff
@@ -0,0 +1,38 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { PageViewTracker } from 'calypso/lib/analytics/page-view-tracker';
+import isAtomicSite from 'calypso/state/selectors/is-site-automated-transfer';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { SiteMetrics } from './main';
+
+export const siteMetrics: PageJS.Callback = ( context, next ) => {
```

Comment:
Can we call this new page `siteMonitoring` instead of `siteMetrics`, and change `client/my-sites/site-metrics` to `client/my-sites/site-monitoring`?

Reply:
Sure, I made the updates